packages/hoppscotch-app/helpers/import-export/export/openapi.ts

```diff
@@ -0,0 +1,213 @@
+/* eslint-disable */
```

Comment:
Not exactly a good sign for whether you are doing things right 🤣

Reply:
was getting a lot of undef and unused variable errors while experimenting. thats why 😂. 
actually this is wip. i had to kind of do some clean up on my pc, so i just commited it as a wip commit. i'll rebase it.